src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2_eink.py

```diff
@@ -38,41 +38,62 @@ def arbeitsl_geld_2_eink_m(
 
     """
     return (
-        _arbeitsl_geld_2_nettoeink_ohne_transfers_m
+        arbeitsl_geld_2_nettoeink_mit_freibetrag_m
         + kind_unterh_erhalt_m
         + unterhaltsvors_m
         + kindergeld_zur_bedarfsdeckung_m
         + kindergeldübertrag_m
     )
 
 
-def _arbeitsl_geld_2_nettoeink_ohne_transfers_m(  # noqa: PLR0913
+def arbeitsl_geld_2_nettoeink_mit_freibetrag_m(
+    arbeitsl_geld_2_nettoeink_ohne_freibetrag_m: float,
+    arbeitsl_geld_2_eink_anr_frei_m: float,
+) -> float:
+    """Net income after deductions for calculation of basic subsistence
+    (Arbeitslosengeld II / Bürgergeld).
+
+    Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
+
+    Parameters
+    ----------
+    arbeitsl_geld_2_nettoeink_ohne_freibetrag_m
+        See :func:`arbeitsl_geld_2_nettoeink_ohne_freibetrag_m`.
+    arbeitsl_geld_2_eink_anr_frei_m
+        See :func:`arbeitsl_geld_2_eink_anr_frei_m`.
+
+    Returns
+    -------
+    Income after taxes, social insurance contributions, and other deductions.
+
+    """
+    return arbeitsl_geld_2_nettoeink_ohne_freibetrag_m - arbeitsl_geld_2_eink_anr_frei_m
+
+
+def arbeitsl_geld_2_nettoeink_ohne_freibetrag_m(
     arbeitsl_geld_2_bruttoeink_m: float,
     eink_st_y_sn: float,
     soli_st_y_sn: float,
     anz_personen_sn: int,
     sozialv_beitr_arbeitnehmer_m: float,
-    arbeitsl_geld_2_eink_anr_frei_m: float,
 ) -> float:
-    """Income (after deduction of taxes, social insurance contributions, and other
-    deductions) for calculation of basic subsistence.
+    """Net income for calculation of basic subsistence (Arbeitslosengeld II /
+    Bürgergeld).
 
     Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
 
     Parameters
     ----------
     arbeitsl_geld_2_bruttoeink_m
         See :func:`arbeitsl_geld_2_bruttoeink_m`.
-    sozialv_beitr_arbeitnehmer_m
-        See :func:`sozialv_beitr_arbeitnehmer_m`.
     eink_st_y_sn
```

Comment:
```suggestion
    eink_st_m_sn
```
Or am I missing something again? :scream:

Reply:
Oh, yes! I was sure this doesn't work for some reason